pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -89,7 +88,8 @@ func (sc *stateCheckpoint) restoreState() error {
 	}
 
 	for containerID, cpuString := range checkpoint.Entries {
-		if tmpContainerCPUSet, err = cpuset.Parse(cpuString); err != nil {
+		tmpContainerCPUSet, err := cpuset.Parse(cpuString)
```

Comment:
I think that one downside of this approach is that we are now initializing `tmpContainerCPUSet` for each iteration through the loop. I'm not sure how large `checkpoint.Entries` can get, so I'm not sure if this can have negative ramifications.

Maybe we could have a middle ground and initialize `tmpContainerCPUSet` once on line 89? That moves the assignment closer to where we use the variable, but has the advantage of only initializing the variable once.

Reply:
Alright I agree and changed it as suggested.